sources/packages/api/src/route-controllers/institution/models/institution.user.res.dto.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  InstitutionUserAuth,
+  InstitutionUserTypeAndRole,
+  User,
+} from "src/database/entities";
+import { InstitutionLocation } from "src/database/entities/institution-location.model";
+import { UserInfo } from "../../../types";
+
+export interface InstitutionUserRespDto {
+  id?: number;
```

Comment:
Is there a scenario where id will be null?

Reply:
No, this will be mandatory field